src/lib/context.js

```diff
@@ -1,134 +1,97 @@
+import { Bundle } from './bundle';
 import { L10nError } from './errors';
-import { format } from './resolver';
-
-const IntlObjects = new WeakMap();
 
 export class Context {
-  constructor(env, langs, resIds) {
+  constructor(langs, resIds) {
```

Comment:
I think it's a bit confusing that `View` and `Context` work with `langs` which are `{ code, src }` objects, and `Bundle` uses `lang` which is a string code (`en-US`).  Any suggestions how to improve this?